k8s/datadog-values.yml

```diff
@@ -43,10 +43,10 @@ agents:
       resources:
         requests:
           cpu: 200m
-          memory: 256Mi
+          memory: 512Mi
         limits:
           cpu: 200m
-          memory: 512Mi
+          memory: 1024Mi
```

Comment:
On our setup it's actually useless to have a different limit from the request. With autopilot, it's the same thing

Reply:
Can you start by increasing request to 512 and leave the limit to 512 as well ?